src/auto-evo/steps/ModifyExistingSpecies.cs

```diff
@@ -157,7 +157,8 @@ public bool RunStep(RunResults results)
         {
             mutation.Item2.OnEdited();
 
-            newMiche.InsertSpecies(mutation.Item2, patch, scores, cache, false, workMemory);
+            if (newMiche.InsertSpecies(mutation.Item2, patch, scores, cache, false, workMemory))
+                scores[mutation.Item2] = 0;
```

Comment:
I'll accept this change as a base, but when I was testing stuff locally I added the score setting *before* calling this method. That's because I thought that the insert species would need to be able to see the species score *before* being able to insert it?

Reply:
I removed this change and did a different fix that in my testing now can run a bunch of generations in the auto-evo exploring tool without errors: https://github.com/Revolutionary-Games/Thrive/pull/5378